File: DiceGame/java/Player.java
public class Player {

  public static void main(String a_args[]) {
  
    System.out.println("Hello Dice World");
    
    DiceGame g = new DiceGame();
    
    if (g.Play()) {
      System.out.println("Win");
    } else {
      System.out.println("Loose");
    }
  
  }

}

File: DiceGame/C#/Player.cs
using System;

namespace DiceGame
{
    class Player
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello Dice World");

            DiceGame g = new DiceGame();

            if (g.Play()) 
            {
                Console.WriteLine("Win");
            } 
            else 
            {
                Console.WriteLine("Loose");
            }

            Console.ReadLine();
        }
    }
}